pkg/mapper/examples/cat_sleep/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/numaproj/numaflow-go/pkg/mapper"
+)
+
+type CatSleep struct {
+}
+
+func (e *CatSleep) Map(ctx context.Context, keys []string, d mapper.Datum) mapper.Messages {
+	time.Sleep(10 * time.Second)
```

Comment:
can we make the duration configurable? thanks!

Reply:
done :)

Reply:
Nice, thanks! I will do that. I was actually wondering if I'd need to go in here all the time and update that.